apps/account-ui/src/components/datalist/EmptyRow.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+} from "@patternfly/react-core";
+
+type EmptyRowProps = {
+  message?: string;
```

Comment:
Looks like `message` is always defined in all places where `EmptyRow` is used, so perhaps this can be required?
```suggestion
  message: string;
```